test/unit/pipeline/test_train_model_sklearn.py

```diff
@@ -40,6 +40,11 @@ def setUp(self) -> None:
             index=pd.date_range(datetime_start, datetime_end, freq="15T")
         )
 
+        self.train_input = TestData.load("reference_sets/307-train-data.csv")
+        self.pj["feature_names"] = self.train_input.columns[
```

Comment:
If features_names are part of our prediction job model this feels bit hacky. In general it would be really nice if we can have a clear data model for our prediction job.